### PR TITLE
Finished class-01 pair-programming assignment

### PR DESCRIPTION
**Pull request by klim95**

### CODE REVIEW

**Comment by klim95** [comment]
So when I drove and Greg navigated, we added the script tags at the bottom of the body tag and linked all of our javascript files, including JQuery. Then we finished the article constructor.
When greg drove and I navigated, we used JQuery to populate the blog articles with the article properties. Then we removed the original template class in the article tag.
We spent around 2 hours pair programming for this assignment.
So basically our biggest challenge was understanding how the javascript was creating each article. Once we understood that, doing JQuery was easier.
